modules/oe_content_entity/modules/oe_content_entity_contact/config/install/core.entity_form_display.oe_contact.general.default.yml

```diff
@@ -0,0 +1,61 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.oe_contact.general.oe_contact_address
+    - field.field.oe_contact.general.oe_contact_email
+    - field.field.oe_contact.general.oe_contact_phone
+    - oe_content_entity_contact.oe_contact_type.general
+  module:
+    - address
+id: oe_contact.general.default
+targetEntityType: oe_contact
+bundle: general
+mode: default
+content:
+  oe_contact_address:
```

Comment:
I'm having some trouble with naming fields, I'll explain: `oe_contact` is the machine name of the entity type, prefixing each field with that would be like prefixing each node field with `node`. Let's use, as a general rule, that we don't prepend field names with entity type IDs.

Still, since these are the fields we ship with, let's use the same convention we used up until now and prepend them with `oe_`.

To summarize let's apply the following changes to all field names, on all entity types:

- `oe_contact_address` -> `oe_address` 
- `oe_contact_email` -> `oe_email` 
- `oe_contact_phone` -> `oe_phone` 
- etc.